### PR TITLE
assets added.

### PR DESCRIPTION
**Pull request by coolnft**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

File: packages/types/src/enums.ts [hunk 1]
```diff
@@ -43,7 +43,6 @@ export enum OracleTypes {
   SimplePriceOracle = "SimplePriceOracle",
   UniswapLpTokenPriceOracle = "UniswapLpTokenPriceOracle",
   UniswapTwapPriceOracleV2 = "UniswapTwapPriceOracleV2",
-  UniswapTwapPriceOracleV3 = "UniswapTwapPriceOracleV3",
```
Comment (carlomazzaferro) [inline packages/types/src/enums.ts]: Any reason why was this removed?

Comment (coolnft) [reply packages/types/src/enums.ts]: no reason, just not needed for this pr.
